src/liquid/place/bulk_create_form.html

```diff
@@ -79,6 +79,15 @@
               <td>Property on primary contact</td>
             </tr>
             {% endfor %}
+
+            {% if contactType.user_role.size > 1 %}
+            <tr>
+              <td>{{ userRoleProperty.friendly_name }}</td>
+              <td>{{ userRoleProperty.required }}</td>
+              <td>Role of the user. Specify multiple roles with <i>space</i>. Allowed roles are <i>{{ contactType.user_role | join: ', ' }}
```

Comment:
```suggestion
              <td>Role of the user. Separate multiple roles with a <i>space</i>. Allowed roles are <i>{{ contactType.user_role | join: ', ' }}
```

Can you have a `space` in a valid CouchDB role?  If so, we should change to `,`.